packages/cactus-plugin-ledger-connector-ethereum/src/main/typescript/web-services/deploy-contract-v1-endpoint.ts

```diff
@@ -88,6 +128,17 @@ export class DeployContractEndpoint implements IWebServiceEndpoint {
   public async handleRequest(req: Request, res: Response): Promise<void> {
     const reqTag = `${this.getVerbLowerCase()} - ${this.getPath()}`;
     this.log.debug(reqTag);
+    const isValid = validateDeployContract(req.body);
+    if (!isValid) {
+      this.log.warn(
+        `Validation failed: ${JSON.stringify(validateDeployContract.errors, null, 2)}`,
```

Comment:
@Harshdev098 Please extract the serialization code into a separate line and don't pretty print the JSON. Also make sure to include the req tag so that it's immediately obvious which endpoint encountered the invalid request.